src/createRollupConfig.ts

```diff
@@ -59,12 +59,7 @@ export async function createRollupConfig(
     // Tell Rollup the entry point to the package
     input: opts.input,
     // Tell Rollup which packages to ignore
-    external: (id: string) => {
-      if (id === 'babel-plugin-transform-async-to-promises/helpers') {
-        return false;
-      }
-      return external(id);
-    },
+    external: external,
```

Comment:
Scratch that... without `useBuiltIns` or `transform-runtime`, no imports are generated for `regenerator-runtime` or `core-js` anyway... 🤔 

That means it might make more sense to use `useBuiltIns` and throw in my suggestion above... Ah, yea I think this solves another issue we've had when regenerator was needed for transpiling actual generator functions (vs. async/await).

Reply:
Thanks for taking the time to review the changes. I'll look at adding automated tests soon.

Also changed it to bundle the polyfills. It adds ~10KB if using an async code, which can be avoided by library authors by increasing the babel build target. Do you think that would warrant a mention in the optimization section of the readme?

One issue with the `external` function as proposed here was that it didn't see the internal dependencies of core-js, leading to core-js's imports ending up in the transpiled code. I solved this by always transpiling core-js with the commonjs() plugin.

Reply:
Replaced `useBuiltIns` with `babel-plugin-polyfill-regenerator`, so `useBuiltIns` is now back to unset/`false`, meaning `core-js` isn't used, so removed it from externals